.github/workflows/main.yml

```diff
@@ -94,9 +109,31 @@ jobs:
           # XCFrameworks are forwards-compatible but not backwards-compatible.
           # The Xcode version we use for this job is that oldest Xcode version that
           # will be able to use these XCFrameworks and the lottie-spm package.
-          xcode: '13.2.1' # Swift 5.5.2
+          xcode: '13.4.1' # Swift 5.6.1
       - name: Build XCFramework
-        run: bundle exec rake build:xcframework
+        run: SKIP_VISION_OS=true bundle exec rake build:xcframework[Lottie-Xcode-${{ matrix.xcode }}]
+      - name: Upload XCFramework
+        uses: actions/upload-artifact@v2
+        with:
+          name: BuildProducts
+          # The xcframework is at the path `.build/archives/Lottie.xcframework.zip`.
+          # GitHub always zips the artifacts before uploading, so if we uploaded the .xframework.zip
+          # directly then it would actually upload a double-zip (a .zip containing our `Lottie.xcframework.zip`).
+          # This is confusing especially since macOS Archive Utility automatially unzips both layers at once.
+          # Instead, we upload the entire archives folder, resulting in an `XCFramework.zip` that unzips
+          # to an `archives` directory containing our `Lottie.xcframework.zip`.
+          path: .build/archives
+
+  build-xcframework-macos-13:
+    name: "Build XCFramework"
+    runs-on: macos-13
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/setup
+        with:
+          xcode: '15.0' # Swift 5.9, first version that includes visionOS SDK.
```

Comment:
We additionally have to build an XCFramework using Xcode 15.0 so that there is an XCFramework that supports visionOS.